content/root/best-practice-guide.textile

```diff
@@ -148,11 +148,11 @@ Whilst we recommend that you use channels to distribute work more evenly across
 
 h3(#embedding-client-id-in-token). Embedding client ID in the token to ensure trust
 
-A clientID serves as an identification for your clients when they are present on a channel. Your clients can set their own client ID, but in that case, there's a possibility that some of your clients can pretend to be someone else. To prevent this, you can embed a client ID in the token issued to your clients from your auth server. Doing so ensures that the client ID is set by your auth server, thus eliminating the chance of the end clients emulating as others.
+A clientID serves as an identification for your clients when they are present on a channel. Your clients can set their own clientID, but in that case, there's a possibility that some of your clients can pretend to be someone else. To prevent this, you can embed a clientID in the token issued to your clients from your auth server. Doing so ensures that the clientID is set by your auth server, thus eliminating the chance of the end clients emulating as others.
```

Comment:
If you're using `clientId` as opposed to `client ID` then it's an attribute / property so should be `@clientId@`. As you will see in https://www.ably.io/documentation/realtime/presence, `clientId` is in that format as opposed to `clientID`.

Reply:
Got it. Fixed [here](https://github.com/ably/docs/commit/6cab87f145513c7cf3551dd6e759c43ff2ed88d8), thanks!